RELEASE.md

```diff
@@ -1,5 +1,12 @@
 # Release notes
 
+## Release 2.0.3
+
+Changes:
+
+* Updated to ArcGIS Runtime 100.9
+* Fixes layout issue affecting some iPhones with notches and a compact width in landscape mode
```

Comment:
Does this fix account for any device with a safe area? The notch isn't the only safe area to account for in iOS. Any iOS device with edge-to-edge rounded corner screens affects the safe area, such as an iPad Pro. Can we make this line more ambiguous?

Reply:
The fix affected an issue that was caused by failing to account for non-safe area when in landscape and compact width, which afaict only happens on iPhone XS and similar. I don't think it can happen on iPad. This wasn't noticed earlier because the only device I had tested that was compact width when in landscape orientation had a rectangle screen.

Reply:
Ah, thank you for the clarification 👍

Reply:
Note that screenshots aren't affected because no screenshots were taken with the affected devices.